src/databricks/labs/ucx/hive_metastore/locations.py

```diff
@@ -154,17 +154,36 @@ def __init__(
         schema: str,
         tables_crawler: TablesCrawler,
         mounts_crawler: 'MountsCrawler',
+        enable_hms_federation: bool = False,
     ):
         super().__init__(sql_backend, "hive_metastore", schema, "external_locations", ExternalLocation)
         self._ws = ws
         self._tables_crawler = tables_crawler
         self._mounts_crawler = mounts_crawler
+        self._enable_hms_federation = enable_hms_federation
 
     @cached_property
     def _mounts_snapshot(self) -> list['Mount']:
         """Returns all mounts, sorted by longest prefixes first."""
         return sorted(self._mounts_crawler.snapshot(), key=lambda _: (len(_.name), _.name), reverse=True)
 
+    def get_dbfs_root(self) -> ExternalLocation | None:
```

Comment:
```suggestion
    def _get_dbfs_root(self) -> ExternalLocation | None:
```

Reply:
Kept it public, it is now called from other classes